Library/Homebrew/compat/sha1.rb

```diff
@@ -1,23 +1,39 @@
 class Formula
   def self.sha1(val)
+    opoo <<-EOS.undent
+      Using SHA-1 is deprecated and will be removed in a future version.
```

Comment:
We've pencilled in the end of September. Don't know if that's worth mentioning more explicitly.